pkg/pipeline/transform/netdb/netdb_test.go

```diff
@@ -0,0 +1,82 @@
+package netdb
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func testingServicesDB() (*ServiceNames, error) {
+	etcProtos, err := os.Open(path.Join("fixtures", "etcProtocols.txt"))
+	if err != nil {
+		return nil, err
+	}
+	defer etcProtos.Close()
+	etcSvcs, err := os.Open(path.Join("fixtures", "etcServices.txt"))
+	if err != nil {
+		return nil, err
+	}
+	defer etcSvcs.Close()
+
+	return LoadServicesDB(etcProtos, etcSvcs)
+}
+
+func TestServicesDB(t *testing.T) {
+	db, err := testingServicesDB()
+	require.NoError(t, err)
+
+	assert.Equal(t, "netbios-dgm", db.ByPortAndProtocolNumber(138, 6))
+	assert.Equal(t, "netbios-dgm", db.ByPortAndProtocolName(138, "tcp"))
+	// verify it also finds service name by protocol alias
+	assert.Equal(t, "netbios-dgm", db.ByPortAndProtocolName(138, "TCP"))
+
+	// verify multiple ports can be associated to the same protocol
```

Comment:
I don't understand this comment. The cases below all have the same port but different protocols. In contrast to the comment that states multiple ports and the same protocol.

Reply:
yeah you are right. Fixing